highfidelity/hifi
# When you unmute using ctrl-m during push to talk, you now see that reflected in the audio icon

When you unmute using ctrl-m during push to talk, you now see that reflected in the audio icon

This pr addresses the following jira case:
https://highfidelity.atlassian.net/browse/BUGZ-76?filter=-1

### Test Plan
1) Go to the Audio settings and enable Push To Talk
2) Close the Audio settings
3) Press and hold T and talk, the other person should be able to hear you.
4) Release T. the other person should not be able to hear you.
5) Press CTRL + M , to unmute the mic. This **does** toggle the little mute display on top of the screen, but after toggling the other person does hear you. 
6) when you press t and release you are back into push to talk

https://highfidelity.manuscript.com/f/cases/22426/
